### PR TITLE
[Api][Admin] Add constraint to product review

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/validation/ProductReview.xml [hunk 1]
```diff
@@ -16,5 +16,13 @@
         <constraint name="Sylius\Bundle\CoreBundle\Validator\Constraints\UniqueReviewerEmail" >
             <option name="groups">sylius_review</option>
         </constraint>
+        <property name="rating">
+            <constraint name="Range">
+                <option name="min">1</option>
+                <option name="max">5</option>
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/CoreBundle/Resources/config/validation/ProductReview.xml:22]: I'm not sure if we should bound the upper limit, and at the very least the value should be configurable.

File: tests/Api/Admin/ProductReviewsTest.php [hunk 2]
```diff
@@ -130,4 +130,33 @@ public function it_updates_a_product_review(): void
...
+
+        /** @var ProductReviewerInterface $review */
+        $review = $fixtures['customer_review'];
+
+        $this->client->request(
+            method: 'PUT',
+            uri: sprintf('/api/v2/admin/product-reviews/%s', $review->getId()),
+            server: $header,
+            content: json_encode([
+                'rating' => -5,
```
Comment (NoResponseMate) [inline tests/Api/Admin/ProductReviewsTest.php]: Use a data provider and also check the other case.

File: src/Sylius/Bundle/CoreBundle/Resources/config/validation/ProductReview.xml (hunk 1 above)
Comment (Wojdylak) [reply src/Sylius/Bundle/CoreBundle/Resources/config/validation/ProductReview.xml:22]: The same constraint we have in `Sylius\Bundle\ApiBundle\Command\Catalog\AddProductReview`, and there's no option to pass parameters to a validation XML file. Therefore, we need to create a custom validator. However, I believe this is out of scope of this task and could we do in a separate PR.

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @Wojdylak!

**Pull request merged**